### PR TITLE
Mention Windows Package Manager as a way to get Ruby

### PR DESCRIPTION
**Pull request by lupino3**

### CODE REVIEW

**Review by hsbt — APPROVED** [review]

File: en/documentation/installation/index.md [hunk 1]
```diff
@@ -217,6 +218,15 @@ $ pkg install runtime/ruby
 However, the third-party tools might be a good way to obtain the
 latest version of Ruby.
 
+### Winget
```
Comment (hsbt) [inline en/documentation/installation/index.md:221]:
```suggestion
### WinGet
```

File: en/documentation/installation/index.md [hunk 2]
```diff
@@ -217,6 +218,15 @@ $ pkg install runtime/ruby
 However, the third-party tools might be a good way to obtain the
 latest version of Ruby.
 
+### WinGet
```
Comment (JuanitoFatas) [inline en/documentation/installation/index.md:221]: Should it be all lowercase per https://docs.microsoft.com/en-us/windows/package-manager/winget/?

CleanShot 2021-09-07 at 21 53 34@2x

Comment (lupino3) [reply en/documentation/installation/index.md:221]: Looking at recent docs (https://devblogs.microsoft.com/commandline/windows-package-manager-1-0/) I think the naming is shifting towards **Windows Package Manager**, so maybe use this name?

The [repo](https://github.com/microsoft/winget-cli/) refers to `winget.exe` as **Windows Package Manager Client**.

Comment (JuanitoFatas) [reply en/documentation/installation/index.md:221]: Windows Package Manager sounds good and clear to folks not yet aware of this tool like me 👍🏼  Very cool.

**Review by JuanitoFatas — APPROVED** [review]
Thanks ❤️

**Comment by lupino3** [comment]
> Thanks ❤️

Thank you for making my update better! :)

File: en/documentation/installation/index.md (hunk 2 above)
Comment (hsbt) [reply en/documentation/installation/index.md]: The lowercase is cli name, not software. I picked from https://github.com/microsoft/winget-cli#manually-update.

**Pull request merged**